ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1143,17 +1372,30 @@ export const RuntimePanel = fp.flow(
               {getWarningMessageContent()}
             </WarningMessage>
            }
-           <FlexRow style={{justifyContent: 'space-between', marginTop: '.75rem'}}>
-             <Link
-               style={{...styles.deleteLink, ...(
-                 (disableControls || !runtimeExists) ?
-                 {color: colorWithWhiteness(colors.dark, .4)} : {}
-               )}}
-               aria-label='Delete Environment'
-               disabled={disableControls || !runtimeExists}
-               onClick={() => setPanelContent(PanelContent.Delete)}>Delete Environment</Link>
-               {!runtimeExists ? renderCreateButton() : renderNextButton()}
-           </FlexRow>
+        {enablePD && selectedCompute === ComputeType.Standard && !runtimeExists && pdExists ?
+            <FlexRow style={{justifyContent: 'space-between', marginTop: '.75rem'}}>
+                <Link
+                    style={{...styles.deleteLink, ...(
+                          (disableControls) ?
+                              {color: colorWithWhiteness(colors.dark, .4)} : {}
+                      )}}
+                    aria-label='Delete Persistent Disk'
+                    disabled={disableControls}
+                    onClick={() => setPanelContent(PanelContent.DeleteUnattachedPd)}>Delete Persistent Disk</Link>
+                {!pdSizeReduced ? renderCreateButton() : renderNextButton()}
```

Comment:
Why are we checking pd size reduction in a delete flow? Shouldn't this button always just say "delete"?

Reply:
It's a create/next button on the same row with the delete button. It has no dependency with the delete button, just about the layout.

Reply:
Can you add a screenshot from your local server here? I'm not understanding this

Reply:
![image](https://user-images.githubusercontent.com/22930983/129825422-5d3a7a6e-90e7-4e84-afb1-270ac46d1773.png)

The CreateButton on the right bottom. It will change to Next Button when we reduce the PD.